src/Filter/UnitRefundFilterInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Filter;
+
+use Sylius\RefundPlugin\Model\UnitRefundInterface;
+
+interface UnitRefundFilterInterface
+{
+    /**
+     * @param UnitRefundInterface[] $units
+     * @param class-string<UnitRefundInterface> $unitRefundClass
+     *
+     * @return UnitRefundInterface[]
```

Comment:
```suggestion
     * @template T of UnitRefundInterface
     *
     * @param UnitRefundInterface[] $units
     * @param class-string<T> $unitRefundClass
     *
     * @return T[]
```

Reply:
phpstan has a problem with that
![image](https://user-images.githubusercontent.com/13316080/225235809-c505383f-b1de-40d1-ab10-c825b844da22.png)